components/service/firefox-accounts/README.md

```diff
@@ -16,6 +23,102 @@ Use gradle to download the library from JCenter:
 implementation "org.mozilla.components:fxa:{latest-version}
 ```
 
+### Start coding
+
+> This tutorial is for version 0.15 of the FxA client.
+
+First you need some OAuth information. Generate a `client_id`, `redirectUrl` and find out the scopes for your application.
+See Firefox Account documentation for that. 
+
+Once you have the OAuth info, you can start adding `FxAClient` to your Android project.
+As part of the OAuth flow your application will be opening up a Web view or open the system browser.
+Currently the SDK does not provide the Web view, you have to write it yourself.
+
+Create a global `account` object: 
+
+```kotlin
+var account: FirefoxAccount? = null
+```
+
+You will need to save state for FxA in your app, this example just uses `SharedPreferences`. We suggest using the Android key store for this data.
```

Comment:
nit: Should we link out to https://developer.android.com/training/articles/keystore here?